src/resmom/stage_func.c

```diff
@@ -1882,6 +1905,7 @@ sys_copy(int dir, int rmtflg, char *owner, char *src, struct rqfpair *pair, int
 							ag0, ag1, ag2_path, ag3_path);
 					}
 				} else {
+					log_err(-1, __func__, "Failed to get UNC path");
```

Comment:
Same suggestion. Passing errno instead of -1 in log_err.

Reply:
WriteFile does not set errno, we need to use GetLastError, so -1 should be good, I think.